src/app/api/dataset.api.ts

```diff
@@ -439,6 +441,25 @@ export class DatasetApi {
             );
     }
 
+    public datasetHashLastBlock(accountName: string, datasetName: string): Observable<DatasetHashLastBlockQuery> {
```

Comment:
Use term "head" everywhere. What is "first" and what is "last" in blockchain is very ambigious.